### ISSUE TITLE
Staging Review finding: Button - Loading imposter component

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#113286]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IIR Team 
**Product:** Veteran Readiness and Employment (VR&E) aka Chapter 31 (28-1900)
**Feature:** Form Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** [User encounters components and patterns that meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (QA), @rsmithadhoc (Accessibility)

### Description

**Submit Application/Sending.../Submitted** can be updated to use [Button - Loading](https://design.va.gov/components/button/#loading). This button also does not have a sufficient contrast ratio between the text and background, which would be fixed by using the VADS button.

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/temporary/apply-vre-form-28-1900/review-and-submit

<img width="457" alt="Image" src="https://github.com/user-attachments/assets/19e23502-71ef-473e-a5b7-b6061e6fbab1" />

</details>

## Recommended action

Use the VADS component.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.4.3 Contrast (Minimum) - Level AA](https://www.w3.org/TR/WCAG22/#contrast-minimum)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
